src/main/java/com/icthh/xm/ms/entity/web/rest/EventResource.java

```diff
@@ -99,8 +100,14 @@ public EventResource(
      */
     @GetMapping("/events")
     @Timed
-    public List<Event> getAllEvents() {
-        return eventService.findAll(null);
+    public List<Event> getAllEvents(EventFilter eventFilter) {
+        List<Event> result;
```

Comment:
seems result variable is redundant. 
and logic could be simplified as: 
```
return eventFilter == null ? eventService.findAll(null) : eventService.findAllByFilter(eventFilter);
```

Reply:
remark applied